code/__DEFINES/ai.dm

```diff
@@ -25,6 +25,8 @@
 
 ///AI flags
 #define STOP_MOVING_WHEN_PULLED (1<<0)
+#define STOP_MOVING (1<<1)
+#define STOP_MOVING_DURING_DO_AFTER (1<<2)
 
```

Comment:
If nothing uses these they'll be removed as redundant eventually, even if they're merged now.
We try and write modular code as a consequence of trying to write clean code. We're not a library, and if you add things that we don't use they're likely to just get poofed in future. It's a bad pattern, and I don't think you should be doing it

Reply:
@LemonInTheDark Removed the defines and associated code, these were here in the first place for more niche usage downstream for making basic mobs climb over tables with a delay.